x-pack/plugins/siem/public/components/page/hosts/host_summary/translations.ts

```diff
@@ -14,8 +14,12 @@ export const NAME = i18n.translate('xpack.siem.hostDetails.name', {
   defaultMessage: 'Name',
 });
 
-export const LAST_BEAT = i18n.translate('xpack.siem.hostDetails.lastBeat', {
-  defaultMessage: 'Last Beat',
+export const FIRST_SEEN = i18n.translate('xpack.siem.hostDetails.firstSeen', {
```

Comment:
[Whoever](https://github.com/elastic/kibana/pull/29311) originally did this PR totally forgot the [i18n message types](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#defining-type-for-message)... ;) This can be addressed in https://github.com/elastic/ingest-dev/issues/364 if desired.

Reply:
I will do it

Reply:
That was probably from the first naive pass at it. Thanks for pointing it out @spong and @XavierM for fixing it.

My bad, my bad....It was me :-(